doc/source/user_guide/ug_running.rst

```diff
@@ -78,8 +78,13 @@ Some hints:
 
 - To change namelist, manually edit the **icepack_in** file
 - To change batch settings, manually edit the top of the **icepack.run** or **icepack.test** (if running a test) file
+- When the run scripts are submitted, the current **icepack_in**, **icepack.settings**, and **env.[machine]** files are copied from the case directory into the run directory.  Users should generally not edit files in the run directory as these are overwritten when following the standard workflow.  **icepack.settings** can be sourced to establish the case values in the login shell.  An alias like the following can be established to quickly switch between case and run directories::
+
+    alias  cdrun 'cd `\grep "setenv ICE_RUNDIR"  icepack.settings | awk "{print "\$"NF}"`'
+    alias cdcase 'cd `\grep "setenv ICE_CASEDIR" icepack.settings | awk "{print "\$"NF}"`'
+
 - To turn on the debug compiler flags, set ``ICE_BLDDEBUG`` in **icepack.setttings** to true
-- To change compiler options, manually edit the Macros file
+- To change compiler options, manually edit the Macros file.  To add user define CPP build flags, users can modify ``ICE_CPPDEFS`` in **icepack.settings**.
```

Comment:
```suggestion
- To change compiler options, manually edit the Macros file.  To add user defined preprocessor macros, users can modify ``ICE_CPPDEFS`` in **icepack.settings**.
```

I'd suggest using standard software engineering terminology, a quick Google search reveals "cppdefs" is not much used outside the climate/weather/environmental modelling communities...

Reply:
aslo I think we can drop "users can", the rest of the bullets are all in the imperative mode.